src/ManagerInterface.php

```diff
@@ -113,13 +115,23 @@ public function revoke(string $itemName, int|Stringable|string $userId): self;
      */
     public function revokeAll(int|Stringable|string $userId): self;
 
+    /**
+     * Returns the ite,s that are assigned to the user via {@see assign()}.
+     *
+     * @param int|string|Stringable $userId The user ID.
+     *
+     * @return array All items directly assigned to the user. The array is indexed by the item names.
+     * @psalm-return ItemsIndexedByName
+     */
+    public function getItemsByUserId(int|Stringable|string $userId): array;
```

Comment:
Why need this method?

Reply:
To get all items (both roles and permissions) assigned to user in one call. Before it was possible to get either roles or permissions only.